src/SFA.DAS.AssessorService.Web/Controllers/OrganisationStandardsController.cs

```diff
@@ -57,18 +57,21 @@ public async Task<IActionResult> Index(int? pageIndex)
                 var organisation = await _organisationsApiClient.GetEpaOrganisation(epaoid);
                 if (organisation != null)
                 {
-                    model.ApprovedStandardsWithVersions =
-                        await _standardsApiClient.GetEpaoRegisteredStandards(
-                            organisation.OrganisationId, pageIndex ?? 1, 10);
+                    model.ApprovedStandardsWithVersions = await _standardsApiClient.GetEpaoRegisteredStandards(organisation.OrganisationId, 1, 10);
 
-                    var financialAssessmentExempt = !string.IsNullOrWhiteSpace(organisation.FinancialReviewStatus) && (organisation.FinancialReviewStatus == ApplyTypes.FinancialReviewStatus.Exempt);
-                    if (!financialAssessmentExempt)
+                    var orgDataFinancialExempt = organisation.OrganisationData?.FHADetails?.FinancialExempt;
+                    var orgDataFinancialDueDate = organisation.OrganisationData?.FHADetails?.FinancialDueDate;
+                    var orgTypeFinancialExempt = organisation.FinancialReviewStatus;
+
+                    if ((!orgDataFinancialExempt.HasValue) || (orgDataFinancialExempt.Value == false))
                     {
-                        var financialDueDate = organisation.OrganisationData?.FHADetails?.FinancialDueDate;
-                        if (null == financialDueDate || (financialDueDate.Value.Date < DateTime.UtcNow.Date))
+                        if ((!orgDataFinancialDueDate.HasValue) || (orgDataFinancialDueDate.Value.Date < DateTime.UtcNow.Date))
                         {
-                            model.FinancialInfoStage1Expired = true;
-                            model.FinancialAssessmentUrl = this.Url.Action("StartOrResumeApplication", "Application");
+                            if (orgTypeFinancialExempt != ApplyTypes.FinancialReviewStatus.Exempt)
```

Comment:
It would be more efficient to check if the organisationtype is exempt, before looking at the organisation data, as these values would only be set for an organisation that is not exempt.

Reply:
Following on from Als comment, I think this can actually be simplified. the `organisation.FinancialReviewStatus` that's returned from the Api, uses the `FinancialReviewStatusHelper.IsFinancialExempt` method to determine the status and then populate either `FinancialExempt` or `Required` so just checking that field alone should be enough ( as it was before the change in this PR ) as at the moment you are re-doing the check that's done in `FinancialReviewStatusHelper` for no real gain.

Reply:
I've made changes for Uma to Test as @anilkdhiman6  is away and the release was scheduled for Monday. Can talk through when you're back Anil.